### ISSUE TITLE
Radiusd will not start after deleting custom SSL Certificate PF 11.1

### ISSUE THREAD

**Issue by PureOxygen-Dev** [inverse-inc/packetfence#6796]
**Describe the bug**
After creating a custom SSL certificate and then deleting it the GUI leaves the radiusd in a failed state.  There is no solution we have found in the GUI to solve this.

**To Reproduce**
1. Create New SSL in Radius | SSL Certificates | New SSL Certificates
2. Complete the various fields and save
3. Update the tls-common profile to use this new certificate (Certificate profile).
4. Restart the radiusd service from the SSH Shell (pfcmd service radiusd restart)
5. Go back to the GUI and delete the created SSL Certificate.
6. Go back to the tls-common profile
7.  Error 1 (deleted certificate is still referenced.)

<img width="1099" alt="image" src="https://user-images.githubusercontent.com/95610834/147210088-a411b4ae-e743-4198-9f94-287b6863ed1a.png">

8. Change the Certificate profile back to the default Radius
9. Restart the radiusd service from the SSH Shell (pfcmd service radiusd restart)
10. Error seen

![image](https://user-images.githubusercontent.com/95610834/147209438-e6faaba1-3571-48e5-b24c-50386127ce53.png)

11. Running in SSH Session /usr/sbin/freeradius -X -d /usr/local/pf/raddb -n auth -fm results in this error

tls: (TLS) Failed reading certificate file "/usr/local/pf/conf/certmanager/radius_default_tls-common.crt"
tls: (TLS) error:0909006C:PEM routines:get_name:no start line
tls: (TLS) error:140DC009:SSL routines:use_certificate_chain_file:PEM lib
rlm_eap_peap: Failed initializing SSL context
rlm_eap (EAP): Failed to initialise rlm_eap_peap

**Expected behavior**
Returning the tls-common back to Certificate Profile radius should result in a working raduisd service.

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Edge Chromium
 - Version:  96.0.1054.62

**Comment by nqb** [comment]
Hello @RHDHV-simon-sutcliffe,

Looks similar to #6580.
Could you try to restart `pfcertmanager` to see if it helps ?

**Comment by PureOxygen-Dev** [comment]
@nqb  Yes that allowed the service to restart without issues.  So seems that needs to be a small update somewhere.
